.github/actions/on_device_tests/action.yaml

```diff
@@ -57,6 +57,7 @@ runs:
             --build_number ${GITHUB_RUN_NUMBER} \
             ${LOADER_PLATFORM:+"--loader_config" "$LOADER_CONFIG"} \
             ${LOADER_PLATFORM:+"--loader_platform" "$LOADER_PLATFORM"} \
+            ${DIMENSION:+"--dimension" "$DIMENSION"} \
```

Comment:
What would happen if DIMENSION isn't defined due to matrix.dimension being null? Would it just ignore this argument?

Reply:
Dimension argument shouldn't get passed at all if DIMENSION is not defined.  There is from bash manual:
 

```
   ${parameter:+word}
          Use Alternate Value.  If parameter is null or  unset,  nothing  is
          substituted, otherwise the expansion of word is substituted.
```